homeworks/05-redux-basic/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Please review which styles are used and remove them.
This is generated by Create React App but still clean up is needed if not used.